examples/network/status.out.static

```diff
@@ -1,602 +1,40 @@
-          "/ip4/131.153.242.59/tcp/30005/p2p/12D3KooWPqxbB3yW8nnzxqh3LkpAKRGt1YeUPfy8t7WhmYdej5R8",
-          "/ip4/66.85.140.10/tcp/30002/p2p/12D3KooWN93Yo85zQ5NgKcBSWTNkGX1SgVNoNCyExpPdj8Av6qW2",
-          "/ip4/66.85.156.101/tcp/31002/p2p/12D3KooWLpJ4ssESgun9yc1MmXUyF7EDYuZWfXpMq9s7BPFrZxaC",
-          "/ip4/131.153.200.7/tcp/30002/p2p/12D3KooWQQTToZP6TL8MbEyFJMQ3H8y7YguJyGE97L4o74AqkUCj",
-          "/ip4/20.245.174.253/tcp/9002/p2p/12D3KooWBmcgkuG2qRq4aD6BDyaytnknGqzZWyw7HZAY31L8rYvs",
-          "/ip4/20.168.196.165/tcp/9002/p2p/12D3KooWJtDiHMyU9Srwj2r5MQABdaErW51rwaJMMTxDZ4XXSNXS",
-          "/ip4/40.76.70.110/tcp/9002/p2p/12D3KooWQJJhtCwJqvQ1CHTW1xD6U9aXceCBmAJTR2GGXyTBsW1g",
-          "/ip4/95.216.224.58/tcp/34648/p2p/12D3KooWGiJwU9ajppAA8R1tFsAezXatNC1fd18VxufnRG7veves",
-          "/ip4/34.94.197.191/tcp/3022/p2p/12D3KooWQ1pjpnq5yAkG3nyawShPwt9WvLjZ6xRxZr86zntztfwK",
-          "/ip4/172.24.72.26/tcp/9200/p2p/12D3KooWQviEyxLL2Sg1r9g6HQzDm8WN8KUwpwL3gPJfieXpmFDL",
-          "/ip4/89.212.17.232/tcp/30002/p2p/12D3KooWEYtZz1pzwvFFUjMRKBckJ7qt6X63PjJVWFrFtezRqC4H",
-          "/ip4/35.193.14.105/tcp/9002/p2p/12D3KooWPJYHjNP5sHPuFQQ83Fa7pAz93oA6oqSWQ5dpbE8MuUQ5",
-          "/ip4/84.255.245.194/tcp/9200/p2p/12D3KooW9tZDg59zv5JA2yi7guggePUpR4mJnriELLKi3RqvC5Y6",
-          "/ip4/20.232.215.170/tcp/9002/p2p/12D3KooWH1cwh3xzGQDT2Dp22Qc3fGGWqn8vYTvm232wp27M71Ft",
-          "/ip4/34.135.151.3/tcp/9002/p2p/12D3KooWEkXQtJ4oxKnWWFucvAp7oBedgCcPhFRwZsy6b1WaJUCU",
-          "/ip4/127.0.0.1/tcp/9200/p2p/12D3KooWDLhy9p7CLK2srzqwFHYRQtrGtRVAe66nJuqyBB61FFbv"
-        ],
-        "host": {
-          "versions": [
-            {
-              "minor": 5,
-              "patch": 2
-            },
-            {
-              "minor": 5,
-              "patch": 1
-            }
-          ]
-        }
-      },
-      "storage": {
-        "last_finalized_round": 1481357
-      }
-    }
-  },
-  "registration": {
-    "last_registration": "0001-01-01T00:00:00Z"
-  }
-}
+=== NETWORK STATUS ===
+Network:                  mainnet
+Node's ID:                8yrQXUf1VIY5lQSCUC9R9FqHn69+SP7OBTuOU+yovvU=
+Core version:             23.0.9
+Consensus:
+  Status:                 ready
+  Version:      7.0.0
+  Chain context:          bb3d748def55bdfb797a2ac53ee6ee141e54cd2ab2dc2375f4a0703a178e6e55
+  Latest Height:          18314588
+  Latest block hash:      c5064d9cc8b6042028b9199749548033286081aebb1eeda46963382c418d605e
+  Latest Epoch:           30511
+  Is validator:           false
+ParaTimes:
+  cipher (000000000000000000000000000000000000000000000000e199119c992377cb):
+    Kind:                 compute
+    Node's Entity ID:     TAv9qXjV4yBphnKLJcNkzois1TLoYUjaRPrMfY58Apo=
+    Status:               ready
+    Storage:              syncing rounds
+    Active version:       3.0.2
+    Avaliable version(s): 2.6.2, 3.0.2
+    Latest round:         1475722 (2024-03-11 12:10:48 +0100 CET)
+    Number of peers:      36
```

Comment:
Seems a bit cluttered. Perhaps something like this would help?

```suggestion

==== ParaTimes ====
cipher (000000000000000000000000000000000000000000000000e199119c992377cb):
  Kind:                 compute
  Node's Entity ID:     TAv9qXjV4yBphnKLJcNkzois1TLoYUjaRPrMfY58Apo=
  Status:               ready
  Storage:              syncing rounds
  Active version:       3.0.2
  Avaliable version(s): 2.6.2, 3.0.2
  Latest round:         1475722 (2024-03-11 12:10:48 +0100 CET)
  Number of peers:      36
```